src/scenes/login/Login.js

```diff
@@ -18,29 +19,81 @@ const styles = StyleSheet.create({
     fontSize: 24,
     marginBottom: 20,
   },
+  form: {
+    fontSize: 24,
+    marginBottom: 20,
+    backgroundColor: 'lime',
+  },
+
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  loginContainer: {
+    width: '80%',
+    alignItems: 'center',
+    padding: 10,
+    elevation: 10,
+    backgroundColor: '#e6e6e6',
+  },
+  textInput: {
+    height: 40,
+    width: '100%',
+    margin: 10,
+    backgroundColor: 'white',
+    borderColor: 'gray',
+    borderWidth: StyleSheet.hairlineWidth,
+    borderRadius: 10,
+  },
```

Comment:
RC: Please create `Login.styles.js` file and move all styles related to Login screeen page to this file